toqito/state_opt/tests/test_bell_inequality_max.py

```diff
@@ -0,0 +1,108 @@
+"""Tests for bell_inequality_max."""
+
+import numpy as np
+import pytest
+import scs
+
+from toqito.state_opt import bell_inequality_max
+
+# Example inputs from the I3322 Bell inequality
+joint_coe = np.array([[1, 1, -1], [1, 1, 1], [-1, 1, 0]])
+a_coe = np.array([0, -1, 0])
+b_coe = np.array([-1, -2, 0])
+a_val_valid = np.array([0, 1])
+b_val_valid = np.array([0, 1])
+expected_value = 0.250
+
+
+@pytest.mark.parametrize(
+    "joint_coe, a_coe, b_coe, a_val, b_val, expected",
+    [(joint_coe, a_coe, b_coe, a_val_valid, b_val_valid, expected_value)],
```

Comment:
I suggest using this test over known Bell inequalities. I did this for I3322, but you could also add the CHSH inequality, etc.

```suggestion
    # Bell I3322 inequality.
    [
        np.array([[1, 1, -1], [1, 1, 1], [-1, 1, 0]]),
        np.array([0, -1, 0]),
        np.array([-1, -2, 0]),
        np.array([0, 1]),
        np.array([0, 1]),
        0.250,
    ],
    # CHSH inequality
```